packages/nostr/src/raw.ts

```diff
@@ -13,7 +15,33 @@ export interface RawEvent {
   sig: string
 }
 
-export function parseRawEvent(data: string): RawEvent {
+interface RawFilters {
+  ids: string[]
+  authors: string[]
+  kinds: number[]
+  ["#e"]: string[]
+  ["#p"]: string[]
+  since: number
+  until: number
+  limit: number
+}
+
+type RawIncomingMessage = ["EVENT", string, RawEvent] | ["NOTICE", string]
```

Comment:
Include `["OK","99d7420bd0505c4f393272a7da1c4ee5127cb5cb74004b45ab39ce8ff6709d55",true,""]` type for event publish response